### PR TITLE
=act split event Unsubscribers.

### PR DESCRIPTION
**Pull request by andreaTP**
Just a plain file split for different purposes unsubscribers.

### CODE REVIEW

**Comment by ktoso** [comment]
Is this split really needed for Akka.js for some reason?
We do like keeping similar classes in the same file.

**Comment by andreaTP** [comment]
Unfortunately ActorClassificationUnsubscriber extends Stash that will be hard to be ported on Scala.js.

At the moment Stash completely rely on JVM specific message queues
https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/actor/Stash.scala#L9

File: akka-actor/src/main/scala/akka/event/EventBusUnsubscribers.scala [hunk 1]
```diff
@@ -72,71 +72,3 @@ private[akka] object EventStreamUnsubscriber {
   }
 
 }
-
```
Comment (patriknw) [inline akka-actor/src/main/scala/akka/event/EventBusUnsubscribers.scala]: Then the remaining thing in this file is `EventStreamUnsubscriber`. Please rename the file to `EventStreamUnsubscriber.scala`

**Comment by patriknw** [comment]
LGTM after file rename

**Comment by andreaTP** [comment]
renamed

**Comment by andreaTP** [comment]
Anything else I can do here?

**Comment by patriknw** [comment]
thanks

**Pull request merged**